evap/grades/forms.py

```diff
@@ -0,0 +1,45 @@
+from django import forms
+from django.conf import settings
+from django.utils.translation import ugettext_lazy as _
+from evap.evaluation.forms import BootstrapMixin
+
+from evap.grades.models import GradeDocument
+
+
+class StrippedCharField(forms.CharField):
+    """
+        CharField that saves trimmed strings
+    """
+
+    def to_python(self, value):
+        super(StrippedCharField, self).to_python(value)
+        if value is None:
+            return None
+        return value.strip()
+
+
+class GradeDocumentForm(forms.ModelForm, BootstrapMixin):
+    description = StrippedCharField(max_length=255)
+    last_modified_time_2 = forms.DateTimeField(label=_("Last modified"), required=False, localize=True)
+    last_modified_user_2 = forms.CharField(label=_("Last modified by"), required=False)
```

Comment:
https://docs.djangoproject.com/en/dev/ref/models/fields/#datefield says that `As currently implemented, setting auto_now to True will cause the field to have editable=False`. and the description for `editable` then explains `If False, the field will not be displayed in the admin or any other ModelForm`.
that means we cannot show the `last_modified_time` field in the form, thus we have to create a new one. also, i would like to keep the current approach for `last_modified_user`, because otherwise we would have a disabled select field there which doesn't make sense.